presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -99,6 +104,16 @@ public void initializeRubix()
         startRubix();
     }
 
+    @PreDestroy
+    public void stopRubix()
+    {
+        if (bookKeeperServer != null) {
+            LocalDataTransferServer.stopServer();
```

Comment:
nit: possibly use closer to ensure we are not getting out of method on first exception.